packages/chat-component-bindings/src/messageThreadSelector.ts

```diff
@@ -193,7 +179,8 @@ export const messageThreadSelector: MessageThreadSelector = createSelector(
       userId,
       showMessageStatus: !isLargeGroup,
       messages: convertedMessages,
-      messageThreadParticipantCount
+      messageThreadParticipantCount,
```

Comment:
Check if this name exists in 1.1.0 api file, if not, please rename it to participantCount, cause this is messageThreadSelector, so the name prefix here is a little redundant

Reply:
This does not exist in `1.1.0`.

Let's rename.